runtime/tests/interpreter/memory_metering_test.go

```diff
@@ -8713,7 +8713,7 @@ func TestInterpretFunctionStaticType(t *testing.T) {
 		_, err := inter.Invoke("main")
 		require.NoError(t, err)
 
-		assert.Equal(t, uint64(1), meter.getMemory(common.MemoryKindFunctionStaticType))
+		assert.Equal(t, uint64(2), meter.getMemory(common.MemoryKindFunctionStaticType))
```

Comment:
Why has this increased? Does this defensive check allocate another static type?

Reply:
Yes, the defensive check gets the static type of the value which the member access is performed on